### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by banzaman**
- Added Linters.
- Added HTML file.
- Added CSS file.
- Added README file.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+.header {
+  display: flex;
+  align-items: center;
+  background-color: #3c3a39;
+  height: auto;
+  justify-content: space-between;
+  flex: 1;
+  padding-bottom: 16px;
+}
```
Comment (julie-ify) [inline style.css:15]: - [ ] Good job displaying the header section with flexbox 👍. However, the `height: auto;` you have here is not what is in the design on Figma and also the height auto makes the header not be like the Figma's. But after I added `height: 63px` and removed the `padding-bottom: 16px` I have a better result.

Your header design:
Loom_XL6V07f64Q

After I fixed it:
Loom_cxrqvfLki6

Here is the changes I made to get that result:
```suggestion
.header {
  display: flex;
  align-items: center;
  background-color: #3c3a39;
  height: 63px;
  justify-content: space-between;
  flex: 1;
}
```

File: style.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+.header a {
+  color: #fff;
+  font-family: "Roboto", sans-serif;
+  margin-left: 24px;
+  text-decoration: none;
+}
```
Comment (julie-ify) [inline style.css:22]: - [ ] Good job adding the required font family according to your design template 👍. However, consider adding other typography as required in your design template. This is to ensure that your design is almost identical to your chosen design template in Figma.

See Figma design: 
Loom_qbZqrJ8ulq

File: style.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  margin-bottom: 12px;
+  margin-right: 24px;
+}
+
+.informativeText p {
+  color: #fff;
+  font-family: "Poppins", sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+}
```
Comment (julie-ify) [inline style.css:72]: - [ ] Consider adding a `margin-bottom: 22px` to this paragraph so as to give a space between this paragraph text and the social icons 💯.

Your design:
Loom_3il0ldLvrz


After I fixed it:
Loom_UdtPxjbbfy

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @banzaman, 

Good job so far!

## Highlight!
- No linter errors ✔️ 
- Good code organization and working app ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi @banzaman  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉🎉🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**